app/routes.js

```diff
@@ -1,9 +1,11 @@
-// const controller = require('./controllers/controller');
 const { healthCheck } = require('./controllers/healthCheck');
 const { listAlbums, getPhotos } = require('./controllers/albums');
-
+const { setUser } = require('./controllers/users');
+const { user_schema } = require('./schemas/user');
```

Comment:
in Node we always use camelCase
only in db columns and in routes we use snake_case